its/plugin/tests/pom.xml

```diff
@@ -49,7 +49,11 @@
     <dependency>
       <groupId>org.sonarsource.sonarqube</groupId>
       <artifactId>sonar-ws</artifactId>
-      <version>6.7</version>
+      <version>7.9</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
```

Comment:
Why that new dependency?

Reply:
Version 6.7 of sonar-ws was transitively providing this dependency and we depend on it. Version 7.9 doesnt export this dependency anymore, so we have to add it

Reply:
Ok.